allocate.go

```diff
@@ -105,11 +106,15 @@ type ExecAllocator struct {
 	initFlags map[string]interface{}
 	initEnv   []string
 
-	wg sync.WaitGroup
+	modifyCmdFunc func(cmd *exec.Cmd)
```

Comment:
Please make this change into its own commit. And explain why the change.

Reply:
See this comment: https://github.com/chromedp/chromedp/pull/1036#issuecomment-1067839839